app/app/urls.py

```diff
@@ -84,7 +84,11 @@
     # personal_tokens
     re_path(r'^token/quickstart/?', ptokens.views.quickstart, name='ptoken_quickstart'),
     re_path(r'^token/faq/?', ptokens.views.faq, name='ptokens_faq'),
-    path('tokens/<str:token_state>/', ptokens.views.tokens, name='token'),
+    path('tokens/redemptions/<int:redemptionId>/', ptokens.views.ptoken_redemption, name='token_redemption'),
+    path('tokens/<int:tokenId>/purchase/', ptokens.views.ptoken_purchases, name='token_purchase'),
+    path('tokens/<int:tokenId>/redemptions/', ptokens.views.ptoken_redemptions, name='token_redemptions'),
+    path('tokens/<int:tokenId>/', ptokens.views.ptoken, name='token'),
+    path('tokens/', ptokens.views.tokens, name='tokens'),
```

Comment:
Should these be /ptoken/<> or /token as they are? 
https://github.com/raid-guild/gitcoinco-web/pull/28/files#diff-b6f3970bc5cdc33271c83021c50a3870
app name is ptokens, not sure how that effects this exactly

Reply:
It doesn't affect, the url and the app name are independent